sbin/pfdhcplistener

```diff
@@ -105,6 +107,9 @@ my $pcap;
 my $net_type;
 my $process;
 
+my $rate_limit_hash = {};
+my $rate_limit_cache = CHI->new( driver => 'Memory', datastore => $rate_limit_hash );
```

Comment:
Just looked into it and if we want to have this available through pfcmd then we can't use the memory storage for obvious reasons..

Now the question is whether we prefer using Redis or the memory storage. I put it in memory for faster access and reduced load on Redis so we'll need to discard this advantage if we want access through pfcmd